nav_3d:
  live_mapper:
    map_pub_topic:  "nav_3d/live_map"
    scan_pub_topic:  "nav_3d/live_scan"
    planar_cloud_topic:  "laser_stitcher/planar_cloud"
    poly_topic:  "nav_3d/robot_footprint_stamped"
    height_topic:  "nav_3d/robot_height"
    robot_base_frame:  "/base_footprint"
    lidar_configuration: 2
    obstacle_algorithm:  "slope" # height or slope
    map_registration:  "scan"
    robot_height_default:  1.0
    robot_height_buffer:   0.05
    robot_height_warning:   0.25
    floor_range:  0.01
    min_obj_dist:  0.0
    max_check_dist: 100.0
    max_robot_reach:  3.0
    scan_res:  3
    map_res:  0.10
    slope_threshold:  0.5
    drivable_height: 0.07
    max_step_height: 0.25
    max_ditch_depth: 0.5 # This should be a positive number. 1.0 will be used as -1.0 m in the z axis
    stale_map_time:  10
    obstacle_decay_factor: 0.60
    obstacle_decay:
      type: "exponential"
      time:  30
    loop_rate: 25
    visualization_tool:
        active: true
        pub_topic: "nav_3d/viz_cloud"
        reset_timer: 5
###  
  robot_state:
    robot_footprint_frame: "/base_footprint"
    halo_radius: 0.1
    halo_density: 10
    height_pub_topic: "nav_3d/robot_height"
    footprint_pub_topic: "nav_3d/robot_footprint"
    frames_to_ignore: ["map", "odom", "left_ur5_tool0_controller", "right_ur5_tool0_controller", "temoto_command_frame", "camera_tail", "camera_anthro_link", "map_centered"]
    loop_rate: 1
